package.json

```diff
@@ -2,13 +2,16 @@
   "name": "xonsh",
   "displayName": "Xonsh IDE",
   "description": "Xonsh language support.",
-  "version": "0.3.3",
+  "version": "0.3.4",
   "publisher": "jnoortheen",
   "icon": "images/icon.png",
   "license": "MIT",
   "engines": {
     "vscode": ">=1.75.0"
   },
+  "extensionKind": [
+    "workspace"
+  ],
```

Comment:
what is the implication if this? does this work when we edit a single file ?

Reply:
yes, this works even though the user open a single file without a workspace opened. The `extensionKind` is only to set the https://code.visualstudio.com/api/advanced-topics/extension-host#preferred-extension-location